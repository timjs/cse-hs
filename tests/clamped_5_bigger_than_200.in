5
assy(begc,cmdk(bwae(dohb(dwcg(dzni,azxt),gbo(cxmp(dgzv(xvb(ddst,akrd),bbxh(dcrj(adpm,difh(deqp(bkfj,emmi),dfrs)),cewh(ceae(ehkf,bgxd),bblr))),coly),eapv)),dmir(cfot,ccmp)),cuja(gv(bxyp(ciqs(lym,ycu(ckeh,cicb(dbaf(bhue(cfxk,coha(cemi,dotj)),aaol),ciin))),crjz),cbvw(ccno(pdz(dqdf,ejzk(bark(donm(aepb(bkcq,cosn),ehzy(duck(cegj,bagk),ckcv(blew,bnju))),bqld(dqbi,bwfx(cwpk(dfsq,kzz),nzz(btpt,poc)))),alda(atms(emvf,eczi),ehtu(dhjb,bvto(bnzg(ayai,aqyx(cofi,eger)),var))))),bpej(cvsv,efhh)),abzj)),afrl)))
bzmj(bjpi(axfz,alue(bytz,dofd(ctuz(cqis,cvzg(tub,moh(dect(aere,douk(dylu,asrs(aygd,dasd))),emjn(dnsk(duwr,arip(buiz(dccg,dhwq),ynb(dzuz(arms,bobz(bsou(abar(bqrz(cqic,bzes),eaiq(bemu,efzh)),bbds(ayhw(ayoc(cdbm,bmyj),epkw),gji)),aeam(doha,buon(dbsk(vrg,eoug),agag(bfum(aygk(cmor,dnxu),drqz),bkoj))))),bxpp(cjdb,bavk)))),mho(aami,enwg(cwox,duii(blgw,awbt(eadz,aubc(cjht,aogk(bhru(bpn(bbqf(ceym,dygb),cmdy),atpz(aslo,dnki)),gtd(daio(cftf(mqv,adbh),clzf),bvyl(oqe,aggl)))))))))))),cahy(jtd,ejny)))),dmzo)
drgd(ejvt(diuw(cgab,cjbf(aosi(bmrl,dnrq(jxi,ahy)),cfmn(asuc(elja,ccsh),cuiz(cfet,eetq)))),epgl),bmtq(ux(bcav(tru(cveb(aues,ddcx),exz),aouz(ccbj,dvxp(ddtf(btlw,dvmh),bkef(dbpy(ahyj,dvre(cthr,aoaq(dwwi(creu,cioh),cwbm))),cyvf)))),bpyr),upu(bdqg(ajsz(dsqt(deqp(dahr(awql,qcz(acsk(dqtx,eprr),emob(bjwx(eama,aoql),datx(azli,aulp(dgbm(axle,anuq),azoj))))),cczr),bqqa(cvkx,cuoe)),ane),bomt),dfcv(bkff(dyph(bgwr,ob),ccow),ejsf(dvif(dwur(cjpk,btid(dzdh,adee)),btry(dhvb(cdyg(arnu,dfyx),dubq),amvo)),cskr)))))
minus(b(a(f(y(y(y(a(baz,y),x(minus(b(a,mult(x(baz,b(y,y(baz,pow(b,plus(minus(b,f),f))))),baz)),x),baz)),y),f(y(minus(baz,plus(minus,b(a(x(b,plus(pow(x,baz),minus(mult(pow,y),a))),x),pow(plus(minus(f,a),f(a,mult(a(a(plus,plus),b),minus))),x)))),f),y(minus(y(plus,a),plus),f))),b),plus(pow(baz(mult,pow(pow(a(x,f(minus(b,minus(plus,f(pow,plus(b,minus)))),pow)),pow(f,pow)),pow(mult(b,y),b(mult(x,f),minus(a,a(pow,a)))))),baz),pow)),a(a,minus)),baz(plus(y(plus(f(y,b),mult),mult),x(f(baz,x),f)),minus))
mult(pow,a(b,a(x(pow(pow(f,x),pow(f(baz(pow,mult(baz,baz)),b),y(a(baz(pow,b(b(baz,b(plus,minus(baz(x(baz,a(x(x,b(mult(y,y(a,b(mult(b(y(baz,x(plus(b,pow),x(plus,mult))),a),minus),y(b,y)))),b(mult(a(a(y,a(f,mult)),x(baz(pow,y),plus)),a(pow(f,pow),a)),minus))),f)),baz(y,a)),baz))),x(plus,y))),pow(plus,minus(baz(pow(mult,y),pow),f(plus,plus)))),pow(minus(minus(minus,mult),b(plus(minus(mult,b(x,minus(mult(minus,f),plus(f,y)))),x),y)),mult(f(x,y),b))))),mult(baz(baz,a),y(y,b(plus,pow(mult,a))))),y)))
