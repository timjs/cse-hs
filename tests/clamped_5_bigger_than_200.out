assy(begc,cmdk(bwae(dohb(dwcg(dzni,azxt),gbo(cxmp(dgzv(xvb(ddst,akrd),bbxh(dcrj(adpm,difh(deqp(bkfj,emmi),dfrs)),cewh(ceae(ehkf,bgxd),bblr))),coly),eapv)),dmir(cfot,ccmp)),cuja(gv(bxyp(ciqs(lym,ycu(ckeh,cicb(dbaf(bhue(cfxk,coha(cemi,dotj)),aaol),ciin))),crjz),cbvw(ccno(pdz(dqdf,ejzk(bark(donm(aepb(bkcq,cosn),ehzy(duck(cegj,bagk),ckcv(blew,bnju))),bqld(dqbi,bwfx(cwpk(dfsq,kzz),nzz(btpt,poc)))),alda(atms(emvf,eczi),ehtu(dhjb,bvto(bnzg(ayai,aqyx(cofi,eger)),var))))),bpej(cvsv,efhh)),abzj)),afrl)))
bzmj(bjpi(axfz,alue(bytz,dofd(ctuz(cqis,cvzg(tub,moh(dect(aere,douk(dylu,asrs(aygd,dasd))),emjn(dnsk(duwr,arip(buiz(dccg,dhwq),ynb(dzuz(arms,bobz(bsou(abar(bqrz(cqic,bzes),eaiq(bemu,efzh)),bbds(ayhw(ayoc(cdbm,bmyj),epkw),gji)),aeam(doha,buon(dbsk(vrg,eoug),agag(bfum(aygk(cmor,dnxu),drqz),bkoj))))),bxpp(cjdb,bavk)))),mho(aami,enwg(cwox,duii(blgw,awbt(eadz,aubc(cjht,aogk(bhru(bpn(bbqf(ceym,dygb),cmdy),atpz(aslo,dnki)),gtd(daio(cftf(mqv,adbh),clzf),bvyl(oqe,aggl)))))))))))),cahy(jtd,ejny)))),dmzo)
drgd(ejvt(diuw(cgab,cjbf(aosi(bmrl,dnrq(jxi,ahy)),cfmn(asuc(elja,ccsh),cuiz(cfet,eetq)))),epgl),bmtq(ux(bcav(tru(cveb(aues,ddcx),exz),aouz(ccbj,dvxp(ddtf(btlw,dvmh),bkef(dbpy(ahyj,dvre(cthr,aoaq(dwwi(creu,cioh),cwbm))),cyvf)))),bpyr),upu(bdqg(ajsz(dsqt(deqp(dahr(awql,qcz(acsk(dqtx,eprr),emob(bjwx(eama,aoql),datx(azli,aulp(dgbm(axle,anuq),azoj))))),cczr),bqqa(cvkx,cuoe)),ane),bomt),dfcv(bkff(dyph(bgwr,ob),ccow),ejsf(dvif(dwur(cjpk,btid(dzdh,adee)),btry(dhvb(cdyg(arnu,dfyx),dubq),amvo)),cskr)))))
minus(b(a(f(y(y(y(a(baz,y),x(minus(b(a,mult(x(9,b(10,y(9,pow(b,plus(minus(20,f),23))))),9)),x),9)),10),f(y(minus(9,plus(minus,b(a(x(20,plus(pow(24,9),minus(mult(pow,10),14))),24),pow(plus(minus(23,14),f(14,mult(a(a(plus,45),20),29))),24)))),23),y(minus(y(45,14),45),23))),20),plus(pow(baz(mult,pow(pow(a(24,f(minus(20,minus(45,f(37,plus(20,29)))),37)),pow(23,37)),pow(mult(20,10),b(mult(24,23),minus(14,a(37,14)))))),9),37)),a(14,29)),baz(plus(y(plus(f(10,20),52),52),x(f(9,24),23)),29))
mult(pow,a(b,a(x(pow(pow(f,x),pow(f(baz(2,mult(baz,15)),4),y(a(baz(2,b(b(15,b(plus,minus(baz(x(15,a(x(10,b(mult(y,y(a,b(mult(b(y(15,x(plus(4,2),x(22,mult))),32),minus),y(4,30)))),b(mult(a(a(30,a(9,40)),x(baz(2,30),22)),a(pow(9,2),32)),41))),9)),baz(30,32)),15))),x(22,30))),pow(22,minus(baz(pow(40,30),2),f(22,22)))),pow(minus(minus(41,40),b(plus(minus(40,b(10,minus(mult(41,9),plus(9,30)))),10),30)),mult(f(10,30),4))))),mult(baz(15,32),y(30,b(22,pow(40,32))))),30)))
